RecoLocalCalo/HcalRecAlgos/test/MahiDebugger.cc

```diff
@@ -158,16 +158,26 @@ class MahiDebugger : public edm::one::EDAnalyzer<edm::one::SharedResources> {
   float chiSq;
   float arrivalTime;
 
-  float pEnergy;    //SOI-1 charge
-  float nEnergy;    //SOI+1 charge
+  float p1Energy;   //SOI-1 charge
+  float p2Energy;   //SOI-2 charge
+  float p3Energy;   //SOI-3 charge
```

Comment:
an appropriate array in this file seems like a better replacement to an explicitly unrolled sequence of calls.

Reply:
These have been replaced with an array **ootEnergy[7]**.